.golangci.yml

```diff
@@ -108,3 +108,8 @@ issues:
     - path: _test\.go
       linters:
         - scopelint
+    # Exclude some staticcheck messages
+    - linters:
+      - staticcheck
+      # See https://github.com/golang/protobuf/issues/1077
```

Comment:
Updated Prometheus also brought `protobuf v1.4.1`, which has `// Deprecated` comment in
https://github.com/golang/protobuf/blob/d04d7b157bb510b1e0c10132224b616ac0e26b17/proto/proto.go#L12
which was introduced in https://go-review.googlesource.com/c/protobuf/+/213901

The change below completely disables the following check:
```
SA1019 | Using a deprecated function, variable, constant or field
```
Is there any better workaround?

Reply:
Extracted to #1308 since it's not related to Prometheus itself.